src/cpu/aarch64/jit_uni_eltwise.cpp

```diff
@@ -202,8 +202,9 @@ struct jit_uni_kernel_t : public jit_uni_eltwise_kernel_t {
 private:
     using TReg = typename cpu_isa_traits<isa>::TReg;
     using TRegS = typename cpu_isa_traits<isa>::TRegS;
-
-    int vlen() { return cpu_isa_traits<isa>::vlen; }
+    int vlen() {
+        return isa == sve_128 ? get_sve_length() : cpu_isa_traits<isa>::vlen;
```

Comment:
Feels like if `isa` is `sve_128`, it should be handled by `cpu_isa_traits` as well and this change shouldn't be needed.

Reply:
I agree with @dzarukin, this isn't really the right place to handle this, but I don't think it's something we should sort out in this PR. But, if we don't, then we need a decent comment here explaining that we are using `sve_128` as a proxy for VLA SVE and a TODO. I also think that it would be clearer to rearrange the logic to check for `asimd` rather than `== sve_128`, because looking at it makes me wonder what happens to `sve_256` and `sve_512`